lisa/tools/ls.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List
+
+from lisa.executable import Tool
+
+
+class Ls(Tool):
+    @property
+    def command(self) -> str:
+        return "ls"
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def path_exists(self, path: str) -> bool:
+        cmd_result = self.run(path, sudo=True, force_run=True)
+        return 0 == cmd_result.exit_code
+
+    def list_dir(self, path: str) -> List[str]:
+        cmd_result = self.node.execute(
+            f"{self.command} -d {path}/*", sudo=True, shell=True
```

Comment:
no shell, and if it needs to support `*`, add the search_pattern as an argument and default is `*`.

Reply:
It is not related with search, without *, it only lists current folder, not sub folder/files

Reply:
Sometime, it needs to list current folder only. How about add an argument `recursive`?

Reply:
recursive is a different concept, it needs all recursive folders under current path. I'll think about it.